core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1671,39 +1741,27 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
   // helper methods
 
   // note: success is indicated by a Future.successful(Map.empty)
-  private def attributeNamespaceCheck(userInfo: UserInfo, attributeNames: Iterable[AttributeName]): Future[Map[String, String]] = {
+  private def attributeNamespaceCheck(userInfo: UserInfo, attributeNames: Iterable[AttributeName]): Map[String, String] = {
```

Comment:
UserInfo is no longer needed here!  Exciting!  Please also remove it from all versions of withAttributeNamespaceCheck.